styles/app.less

```diff
@@ -107,3 +107,22 @@ a:hover {
 		margin: 10px 0;
 	}
 }
+
+.filter {
+	.input-group-append {
+		position: relative;
+
+		a.clear-filter {
+			position: absolute;
+			top: 0;
+			left: -38px;
+			z-index: 1000;
+			color: #b1b1b1;
```

Comment:
if this is some color from bootstrap, I'd use the variable here.

Reply:
No, actually this is not, but it might be.